<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListItems</class>
 <widget class="QWidget" name="ListItems">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>МОЛ:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LComboBox" name="molBox"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Адрес:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LComboBox" name="addrBox"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Пользователь:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LComboBox" name="userBox"/>
     </item>
     <item>
      <widget class="QToolButton" name="moveButton">
       <property name="text">
        <string>Перемещения</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">lcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
